webapp/channels/src/components/select_team/select_team.tsx

```diff
@@ -289,7 +290,17 @@ export default class SelectTeam extends React.PureComponent<Props, State> {
                                 defaultMessage='Teams you can join: '
                             />
                         </h4>
-                        <LearnAboutTeamsLink/>
+                        <ExternalLink
+                            location='learn_about_teams'
+                            href='https://mattermost.com/pl/mattermost-academy-team-training'
+                            className='LearnAboutTeamsLink'
```

Comment:
does this link need aria label or will it take it from inner text element? if so, should the icon need the aria-hidden?

Reply:
i added hidden to the icon, but it reads out the text correctly instead of the link